src/commands.ts

```diff
@@ -184,6 +190,14 @@ function archive () {
 
 }
 
+function insertNewLine () {
+
+  Utils.isShiftEnter('set');
```

Comment:
Will this set the flag to true or false? this 'set' thing is actually a "toggle", which is weird.